openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx

```diff
@@ -191,8 +193,10 @@ const ExplorePage: FunctionComponent = () => {
       searchQuery({
         query: searchQueryParam,
         searchIndex,
-        queryFilter: advancesSearchQueryFilter,
-        postFilter: elasticsearchPostFilterQuery,
+        queryFilter: getCombinedQueryFilterObject(
+          elasticsearchQueryFilter as unknown as QueryFilterInterface,
+          advancesSearchQueryFilter as unknown as QueryFilterInterface
+        ),
```

Comment:
Why are we typecasting?

Reply:
Both query filter objects have type as `Record<string, unknown>`, so here `unknown` will not allow us to directly access the properties, that's why we have to first typecast it into some type and then we can access the properties.
I will add the comment in the code as well.